cirq-core/cirq/qis/channels_test.py

```diff
@@ -80,14 +96,71 @@ def test_kraus_to_choi(kraus_operators, expected_choi):
     ),
 )
 def test_operation_to_choi(channel):
-    """Verifies that cirq.choi correctly computes the Choi matrix."""
+    """Verifies that cirq.operation_to_choi correctly computes the Choi matrix."""
     n_qubits = cirq.num_qubits(channel)
     actual = cirq.operation_to_choi(channel)
     expected = expected_choi(channel)
     assert np.isclose(np.trace(actual), 2 ** n_qubits)
     assert np.all(actual == expected)
 
 
-def test_choi_on_completely_dephasing_channel():
-    """Checks that cirq.choi returns the right matrix for the completely dephasing channel."""
+def test_choi_for_completely_dephasing_channel():
+    """Checks cirq.operation_to_choi on the completely dephasing channel."""
     assert np.all(cirq.operation_to_choi(cirq.phase_damp(1)) == np.diag([1, 0, 0, 1]))
+
+
+@pytest.mark.parametrize(
+    'kraus_operators, expected_channel_matrix',
+    (
```

Comment:
Most of the repeated constants were the Kraus operators of the completely depolarizing channel. I changed the code to use a short cirq expression to generate those instead of hardcoding. I also changed Kraus operators in another test case. The remaining constants are either different or very short (e.g. `np.eye(2)`). This cuts away almost all duplication (the Choi and channel matrices were of course different all along) while keeping all inputs local so the reader does not need to scroll to examine correctness.